gateway/src/main/java/io/zeebe/gateway/impl/job/LongPollingActivateJobsRequest.java

```diff
@@ -23,6 +24,7 @@
   private final StreamObserver<ActivateJobsResponse> responseObserver;
   private final String jobType;
   private final int maxJobsToActivate;
+  private Duration longPollingTimeout;
```

Comment:
It seems that the timeout is not changed.

```suggestion
  private final Duration longPollingTimeout;
```